client/src/ui/replay/DraftPicker.vue

```diff
@@ -14,38 +14,33 @@
       </div>
       <div v-else class="no-picks">You don't have any picks (yet)</div>
     </div>
-    <div class="pool">
-      <div class="pool-title">Drafted cards</div>
-      <div class="pool-cnt">
-        <CardView
-            v-for="cardId in currentPool"
-            :key="cardId"
-            :card="draftStore.getCard(cardId)"
-            class="card"
-            />
-      </div>
-    </div>
+    <DeckBuilderMain class="pool" :horizontal="true" />
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import { authStore } from '../../state/AuthStore';
 import { draftStore, DraftStore } from '../../state/DraftStore';
-import { replayStore, ReplayStore } from '../../state/ReplayStore';
+import { replayStore } from '../../state/ReplayStore';
 
 import CardView from './CardView.vue';
-import { CardPack, DraftSeat, DraftCard } from '../../draft/DraftState';
-import { TimelineEvent } from '../../draft/TimelineEvent';
-import { checkNotNil } from '../../util/checkNotNil';
+import { CardPack, DraftSeat } from '../../draft/DraftState';
 import { fetchEndpoint } from '../../fetch/fetchEndpoint';
 import { routePick } from '../../rest/api/pick/pick';
 import { delay } from '../../util/delay';
+import DeckBuilderMain from "../deckbuilder/DeckBuilderMain.vue";
+import { deckBuilderStore } from "../../state/DeckBuilderModule";
 
 export default Vue.extend({
 
   components: {
-    CardView
+    CardView,
+    DeckBuilderMain,
+  },
+
+  created() {
+    deckBuilderStore.setSelectedSeat(this.currentSeat.position);
```

Comment:
Do we need to update this if the current seat changes? I guess it can't practically change.

Reply:
Yeah, since it can't change in practice I'm not bothering to figure out where I'd need to update it.